hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseOptions.java

```diff
@@ -137,4 +138,28 @@ public static synchronized HbaseOptions instance() {
                     null,
                     ""
             );
+
+    public static final ConfigOption<Integer> HBASE_VERTEX_TABLE_REGION_NUM =
+            new ConfigOption<>(
+                    "hbase.hbase_vertex_table_region_num",
+                    "The number of partitions of the HBase vertex table, it can be set to the number of pre-split regions.",
+                    nonNegativeInt(),
+                    1
+            );
+
+    public static final ConfigOption<Integer> HBASE_EDGE_TABLE_REGION_NUM =
+            new ConfigOption<>(
+                    "hbase.hbase_edge_table_region_num",
+                    "The number of partitions of the HBase edge table, it can be set to the number of pre-split regions.",
```

Comment:
we also could enhance the comment, add a basic reference for user to config (like 4X~10X depends with vertex amount), also add a **uncomment** config in [hugegraph.properties](https://github.com/hugegraph/hugegraph/blob/b591f7d8b3d67438fca98e62da5cdb646d9b03f2/hugegraph-dist/src/assembly/static/conf/graphs/hugegraph.properties#L68)